data/json/items/vehicle/cables.json

```diff
@@ -18,6 +18,25 @@
     "use_action": "CABLE_ATTACH",
     "flags": [ "CABLE_SPOOL" ]
   },
+  {
+    "type": "TOOL",
+    "id": "tow_cable",
+    "name": "tow cable",
+    "description": "A tow cable, attached to a vehicle, it could be used to pull another vehicle.",
```

Comment:
```suggestion
    "description": "An extremely heavy duty tow cable made from thick steel wire.  If attached to a vehicle, it could be used to pull another vehicle of any weight.",
```